dummy_robot/dummy_map_server/src/dummy_map_server.cpp

```diff
@@ -64,7 +66,8 @@ int main(int argc, char * argv[])
     msg->data[(++center) % (msg->info.width * msg->info.height)] = 100;
     msg->data[(++rhs) % (msg->info.width * msg->info.height)] = 0;
 
-    msg->header.stamp = rclcpp::Time::now();
+    rclcpp::Clock ros_clock(RCL_ROS_TIME);
+    msg->header.stamp = ros_clock.now();
```

Comment:
Unlike in https://github.com/ros2/geometry2/pull/49, this code doesn't seem to include the node or time source in here anywhere, is that intentional?